src/core/syntax.ml

```diff
@@ -198,6 +198,8 @@ type p_command_aux =
   | P_definition of Terms.expo * bool * ident * p_arg list * p_type option
                   * p_term
   (** Definition of a symbol (unfoldable). *)
+  | P_inductive of Terms.expo * ident * p_type * p_term list   (* @Moi *)
```

Comment:
Shouldn't you have (ident * p_term) list instead of p_term list? (You need the names of constructors.) And p_type should optional (p_term option).

Reply:
You right but I have a problem to define the good pretty in pretty.ml.

Reply:
Now it's OK.